conn2res/reservoir.py

```diff
@@ -9,8 +9,223 @@
 import numpy as np
 import numpy.ma as ma
 from numpy.linalg import (pinv, matrix_rank)
+from scipy.linalg import eigh
+from bct.algorithms.clustering import get_components
+from bct.algorithms.distance import distance_bin
+
+from .iodata import load_file
+from .coding import get_modules
+
+
+class Conn:
+    """
+    Class that represents a connectivity matrix representing either weighted
+    or unweighted connectivity data
+
+    Attributes
+    ----------
+    # TODO
+
+    Methods
+    ----------
+    # TODO
+    """
+
+    def __init__(self, subj_id=0):
+        # load connectivity data
+        self.w = load_file('connectivity.npy')
+
+        # select one subject
+        self.w = self.w[:, :, subj_id]
+
+        # number of all active nodes
+        self.n_nodes = len(self.w)
+
+        # indexes of set of active nodes
+        self.idx_node = np.full(self.n_nodes, True)
+
+    def scale_and_normalize(self):
```

Comment:
I thought that scaling and normalizing is useful to do in 2 separate steps, so I wrote methods for them separately but this function can also perform both operations at the same time.

Also, these operations are not called during the initialization of the Conn class so that these steps can be skipped, for instance, in the mssn example/workflow.